main.cpp

```diff
@@ -373,7 +326,34 @@ Thing 4) Ranch
     1) sell ranch
     2) open the doors
     3) free the animals
-/*
+*/
+struct ranch
+{
+  int numberOfAnimals;
+  float sizeOfRanch;
+  double amountOfProfitYear;
+  int numberOfDoors;
+  int numberOfTools;
+
+  struct ranchMainA
+  {
+    ranchMainA();
+
+    std::string name = "Zeleia";
+    double height = 223.12;
+    double width = 422.64;
+    std::string directorName = "Asli Yalcin";
+    double todayValue = 354.644;
+
+    void findCustomer(int totalTarget = 6);
+    void clearRanch(int numberOfParts = 4, bool getCleanPart3 = true);
+    void defineValue(double initialPrice = 271.640);
+  };  
+
+  void sellRanch(std::string customerName = "Leyla", int age = 63);
+  void openTheDoors(int numbersOfDoors = 2);
+  void freeTheAnimals(int animalNumbers = 66);
+};
```

Comment:
Fix the following:
- Type Name capitalization
- Indentation relative to the `{ }`
- [ ] revise